ceph-releases/kraken/ubuntu/16.04/daemon/watch_mon_health.sh

```diff
@@ -2,7 +2,6 @@
 set -e
 
 function watch_mon_health {
-  log "checking for zombie mons"
```

Comment:
this is also log just after in the while loop so at first it print it twice:
2017-03-30 07:44:57  /entrypoint.sh: checking for zombie mons
2017-03-30 07:44:57  /entrypoint.sh: checking for zombie mons

and while instruction can't fail.